.scripts/prepare_example.py

```
@@ -24,7 +25,13 @@
     # Change the file extension to .cpp
     base = os.path.splitext(file_path)[0]
     new_file_path = base + '.cpp'
-    os.rename(file_path, new_file_path)
+    shutil.copyfile(file_path, new_file_path)
+
+    with open(new_file_path, 'r') as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)